adm_program/system/classes/Email.php

```diff
@@ -174,6 +174,65 @@ public function addRecipient($address, $name = '')
         return true;
     }
 
+    /**
+     * Add the name and email address of the given user id to the email as a normal recipient. If the system setting
+     * **mail_send_to_all_addresses** is set than all email addresses of the given user id will be added.
+     * @param int $userId Id of an user who should be the recipient of the email.
+     * @return Returns true if recipients could be added to the email.
+     * @throws AdmException in case of errors. exception->text contains a string with the reason why no recipient could be added.
+     *                     Possible reasons: MSG_NO_VALID_RECIPIENTS
+     */
+    public function addRecipientByUserId($userId)
```

Comment:
as it could be more than one address it should be `addRecipientsByUserId` (plural s)